src/main/scala/io/qbeast/spark/delta/OTreeIndex.scala

```diff
@@ -81,9 +83,18 @@ case class OTreeIndex(index: TahoeLogFileIndex) extends FileIndex {
     }.toArray
     val stagingStats = stagingFiles
     val fileStats = qbeastFileStats ++ stagingStats
-
+    val sc = SparkSession.active.sparkContext
+    val execId = sc.getLocalProperty(SQLExecution.EXECUTION_ID_KEY)
+    val pfStr = partitionFilters.map(f => f.toString).mkString(" ")
+    logInfo(s"OTreeIndex partition filters (exec id ${execId}): ${pfStr}")
+    val dfStr = dataFilters.map(f => f.toString).mkString(" ")
+    logInfo(s"OTreeIndex data filters (exec id ${execId}): ${dfStr}")
+    val allFilesCount = snapshot.allFiles.count
+    val nFiltered = allFilesCount - fileStats.length
+    val filteredPct = (((nFiltered * 1.0) / allFilesCount) * 100.0).round
```

Comment:
I think it would be best to have percentage with at least 2 decimal digits (e.g. 0.23% or 53.23%). 

You can use the f instead of the s to format properly numbers. For instance:

```
val filteredPct = (nFiltered * 100.0) / allFilesCount
val filteredMsg = f"${nFiltered} of ${allFilesCount} (${filteredPct}%.2f%%)"
```

Reply:
Ok, changed to print two decimals.